.meteor/release

```diff
@@ -1 +1 @@
-METEOR@1.2.1
+METEOR@1.2.2-cdn-url
```

Comment:
Sorry, I don't know where this came from. Probably a `meteor update` gone too far. I'm removing it.